src/modules/PublicLab.CustomInsert.js

```diff
@@ -67,7 +67,13 @@ module.exports = function CustomInsert(_module, wysiwyg) {
     }
   });
   const builder = require("./PublicLab.CustomInsert.Template.js");
-  $('.wk-commands').append('<a class="woofmark-command-insert btn btn-outline-secondary" data-toggle="Insert" title="Custom Insert"><i class="fa fa-tags"></i></a>');
+
```

Comment:
Good job, just a little change here, line 70 shouldn't be a blank line. That's where the code additions should start.

Reply:
Thanks! Is the new commit better?

Reply:
Looks good to me! 
Well done 🥳